src/Reference/ContainsOne.php

```diff
@@ -82,24 +83,21 @@ public function init(): void
      */
     protected function getDefaultPersistence($model)
```

Comment:
can we drop this?

Reply:
It is used in sub-classes so I would not touch it

Reply:
I think it should be removed

Reply:
You can work on it if you like

Reply:
on Tuesday we start new icescream shop, not that much time now ;-)

I would like this to be solved - but it is not directly related to this PR, thus is has to be not solved now